handlers/holiday-stop-processor/src/main/scala/com/gu/holidaystopprocessor/HolidayCreditProduct.scala

```diff
@@ -11,73 +11,91 @@ import com.gu.zuora.subscription.{CreditProduct, RatePlan}
   */
 object HolidayCreditProduct {
 
-  final val ProductRatePlanChargeName: String = "Holiday Credit"
-
-  object Prod {
-
-    val GuardianWeekly: CreditProduct = CreditProduct(
+  private[holidaystopprocessor] case class HolidayCreditProductForStage(
+      GuardianWeekly: CreditProduct,
+      Voucher: CreditProduct,
+      VoucherObserverOnly: CreditProduct,
+      DigitalVoucher: CreditProduct,
+      DigitalVoucherObserverOnly: CreditProduct,
+      HomeDelivery: CreditProduct,
+      HomeDeliveryObserverOnly: CreditProduct,
+      NationalDelivery: CreditProduct,
+  )
+
+  private[holidaystopprocessor] val Prod: HolidayCreditProductForStage = HolidayCreditProductForStage(
+    GuardianWeekly = CreditProduct(
       productRatePlanId = "2c92a0076ae9189c016b080c930a6186",
       productRatePlanChargeId = "2c92a0086ae928d7016b080f638477a6",
-      ProductRatePlanChargeName,
-    )
-
-    val Voucher: CreditProduct = CreditProduct(
+    ),
+    Voucher = CreditProduct(
       productRatePlanId = "2c92a0117468816901748bdb3a8c1ac4",
       productRatePlanChargeId = "2c92a0117468816901748bdb3aab1ac6",
-      ProductRatePlanChargeName,
-    )
-
-    val DigitalVoucher: CreditProduct = CreditProduct(
+    ),
+    VoucherObserverOnly = CreditProduct(
+      productRatePlanId = "8a12994695dbb387019610e3b64b4985",
+      productRatePlanChargeId = "8a12994695dbb387019610e3b675498b",
+    ),
+    DigitalVoucher = CreditProduct(
       productRatePlanId = "2c92a0fe750b35d001750d4522f43817",
       productRatePlanChargeId = "2c92a0fe750b35d001750d4523103819",
-      ProductRatePlanChargeName,
-    )
-
-    val HomeDelivery: CreditProduct = CreditProduct(
+    ),
+    DigitalVoucherObserverOnly = CreditProduct(
+      productRatePlanId = "8a12817595db977f019610de92e23fde",
+      productRatePlanChargeId = "8a12817595db977f019610de93323fe9",
+    ),
+    HomeDelivery = CreditProduct(
       productRatePlanId = "2c92a00f7468817d01748bd88f0d1d6c",
       productRatePlanChargeId = "2c92a00f7468817d01748bd88f2e1d6e",
-      ProductRatePlanChargeName,
-    )
-
-    val NationalDelivery: CreditProduct = CreditProduct(
+    ),
+    HomeDeliveryObserverOnly = CreditProduct(
+      productRatePlanId = "8a12994695dbb387019610e2556647ac",
+      productRatePlanChargeId = "8a12994695dbb387019610e2559747ae",
+    ),
```

Comment:
![image](https://github.com/user-attachments/assets/f822e924-4ffd-47d7-a73d-68d7bd98c2e3)
```suggestion
    HomeDeliveryObserverOnly = CreditProduct(
      productRatePlanId = "8a12994695dbb387019610e2556647ac",
      productRatePlanChargeId = "8a12994695dbb387019610e2559747ae",
    ),
```